docker/nginx/conf.d/client.conf

```diff
@@ -308,4 +291,7 @@ server {
 
 		proxy_pass http://127.0.0.1/$uri?attachment=true&$args;
 	}
+
+	# include custom locations, specific to the server
+	include /etc/nginx/conf.d/locations/*;
```

Comment:
am I missing something here? are we doing `locations` or `server-override`

Reply:
```suggestion
	include /etc/nginx/conf.d/server-override/*;
```